CHANGELOG.md

```diff
@@ -13,6 +13,18 @@
 - Add `pdata.AttributeMap.RemoveIf`, which is a more performant way to remove multiple keys (#4914)
 - Add `pipeline` key with pipeline identifier to processor loggers (#4968)
 
+### 🚩 Deprecations 🚩
+
+- Change behavior of oneof numeric value field accessors (#4950)
```

Comment:
How does the behavior change?  Do these also panic if not carefully accessed?  That would be important to call out here.

Reply:
Yes, the same behavior as the AttributeValue accessors. I'll update it once we agree on the API in https://github.com/open-telemetry/opentelemetry-collector/pull/4909